boards/arm/qemu_cortex_m0/nrf_timer_timer.c

```diff
@@ -74,13 +74,13 @@ int z_clock_driver_init(struct device *device)
 
 	ARG_UNUSED(device);
 
-	clock = device_get_binding(DT_INST_0_NORDIC_NRF_CLOCK_LABEL "_16M");
+	clock = device_get_binding(DT_INST_0_NORDIC_NRF_CLOCK_LABEL);
 	if (!clock) {
 		return -1;
 	}
 
 	/* turn on clock in blocking mode. */
```

Comment:
Shouldn't you delete this comment too? The `(void*)1` that is removed by this patch used to have blocking semantics, but it no longer does.

Reply:
you right that comment may be redundant. In fact, `clock_control_on` should be blocking (even if it isn't now).

Reply:
comment removed